web/js/webThrottle.js

```diff
@@ -735,7 +735,7 @@ var defineTurnoutsRoutes = function(listTurnoutsRoutes) {
 		bodyFrameInner.append(trCell);
 		trCell.append($('<div>').text(item.data.name + ((item.data.userName) ? ' - ' + item.data.userName : '')).addClass('trName'));
 		trCell.append($('<div>').text('').addClass('trStatus').attr('id',encodeId(item.data.name)).attr('state', -1).on('click', function(event) {trChangeStatus(event, item.type, item.data.name);}));
-		$jmri.getJMRI(item.type == 'turnout' ? 'turnout' : 'route', item.data.name);
+		$jmri.getJMRI(item.type == 'turnouts' ? 'turnout' : 'route', item.data.name);
```

Comment:
Rather than switch from singular to plural, I suggest using 
`item.type.startsWith('turnout')`
to support either form. In my testing, this prevents the sending of the incorrect type and subsequent error:
`"Error 404 - Object type route named "LT416" not found.`

Reply:
OK